build.sh

```diff
@@ -1,110 +0,0 @@
-#!/bin/sh
```

Comment:
was this for local running of plugin? 
do you have in the readme the docker way of getting it running?

Reply:
This script has been obsolete for many years. We've certainly never used it since migrating to AWS.